<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SceneEdit</string>
  </property>
  <property name="animated" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="m_centralwidget" >
   <layout class="QVBoxLayout" >
    <item>
     <widget class="QSplitter" name="splitter" >
      <property name="orientation" >
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" native="1" name="widget" />
      <widget class="QWidget" native="1" name="widget_2" >
       <layout class="QVBoxLayout" >
        <item>
         <widget class="QSplitter" name="splitter_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="Shell" name="textEdit" >
           <property name="toolTip" >
            <string/>
           </property>
           <property name="whatsThis" >
            <string/>
           </property>
          </widget>
          <widget class="QTabWidget" name="tabWidget_2" >
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="currentIndex" >
            <number>0</number>
           </property>
           <widget class="QWidget" name="tab_3" >
            <attribute name="title" >
             <string>Info</string>
            </attribute>
            <layout class="QVBoxLayout" >
             <item>
              <widget class="QGroupBox" name="groupBox" >
               <property name="title" >
                <string>Selected node</string>
               </property>
               <layout class="QGridLayout" >
                <item row="0" column="0" >
                 <widget class="QLabel" name="label" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>Name:</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1" >
                 <widget class="QLabel" name="_nameLabel" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0" >
                 <widget class="QLabel" name="label_2" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>Center:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1" >
                 <widget class="QLabel" name="_centerLabel" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" >
                 <widget class="QLabel" name="label_3" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>Radius:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1" >
                 <widget class="QLabel" name="_radiusLabel" >
                  <property name="sizePolicy" >
                   <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="text" >
                   <string>TextLabel</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tab_4" >
            <attribute name="title" >
             <string>Properties</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="m_menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File" >
    <property name="tearOffEnabled" >
     <bool>true</bool>
    </property>
    <property name="title" >
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew" />
    <addaction name="actionOpen" />
    <addaction name="separator" />
    <addaction name="actionSave" />
    <addaction name="actionSaveAs" />
    <addaction name="separator" />
    <addaction name="actionExit" />
   </widget>
   <widget class="QMenu" name="menuEdit" >
    <property name="tearOffEnabled" >
     <bool>true</bool>
    </property>
    <property name="title" >
     <string>Edit</string>
    </property>
    <addaction name="actionUndo" />
    <addaction name="actionRedo" />
    <addaction name="separator" />
    <addaction name="actionCut" />
    <addaction name="actionCopy" />
    <addaction name="actionPaste" />
   </widget>
   <addaction name="menu_File" />
   <addaction name="menuEdit" />
  </widget>
  <widget class="QStatusBar" name="m_statusBar" />
  <widget class="QDockWidget" name="_treeDock" >
   <property name="allowedAreas" >
    <set>Qt::LeftDockWidgetArea|Qt::NoDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle" >
    <string>Scene</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="leftMargin" >
      <number>9</number>
     </property>
     <property name="topMargin" >
      <number>9</number>
     </property>
     <property name="rightMargin" >
      <number>9</number>
     </property>
     <property name="bottomMargin" >
      <number>9</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="tabPosition" >
        <enum>QTabWidget::North</enum>
       </property>
       <property name="tabShape" >
        <enum>QTabWidget::Rounded</enum>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <property name="elideMode" >
        <enum>Qt::ElideNone</enum>
       </property>
       <widget class="QWidget" name="tab" >
        <attribute name="title" >
         <string>Graph</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>6</number>
         </property>
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>9</number>
         </property>
         <property name="rightMargin" >
          <number>9</number>
         </property>
         <property name="bottomMargin" >
          <number>9</number>
         </property>
         <item>
          <widget class="QTreeWidget" name="_treeWidget" >
           <property name="selectionMode" >
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="selectionBehavior" >
            <enum>QAbstractItemView::SelectItems</enum>
           </property>
           <property name="columnCount" >
            <number>1</number>
           </property>
           <column>
            <property name="text" >
             <string>Nodes</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="_treeSlider" >
           <property name="maximum" >
            <number>1</number>
           </property>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition" >
            <enum>QSlider::TicksBelow</enum>
           </property>
           <property name="tickInterval" >
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="_standardBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew" />
   <addaction name="actionOpen" />
   <addaction name="actionSave" />
   <addaction name="separator" />
   <addaction name="actionCut" />
   <addaction name="actionCopy" />
   <addaction name="actionPaste" />
   <addaction name="separator" />
   <addaction name="actionUndo" />
   <addaction name="actionRedo" />
  </widget>
  <widget class="QToolBar" name="_navigationBar" >
   <property name="windowTitle" >
    <string>toolBar_2</string>
   </property>
   <property name="orientation" >
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="iconSize" >
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionHome" />
   <addaction name="actionZoom" />
  </widget>
  <action name="actionNew" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/filenew.png</iconset>
   </property>
   <property name="text" >
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/fileopen.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>Save</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs" >
   <property name="enabled" >
    <bool>true</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/filesaveas.png</iconset>
   </property>
   <property name="text" >
    <string>Save As</string>
   </property>
  </action>
  <action name="actionExit" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/exit.png</iconset>
   </property>
   <property name="text" >
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/undo.png</iconset>
   </property>
   <property name="text" >
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/redo.png</iconset>
   </property>
   <property name="text" >
    <string>Redo</string>
   </property>
  </action>
  <action name="actionCut" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/editcut.png</iconset>
   </property>
   <property name="text" >
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/editcopy.png</iconset>
   </property>
   <property name="text" >
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/32x32/editpaste.png</iconset>
   </property>
   <property name="text" >
    <string>Paste</string>
   </property>
  </action>
  <action name="actionHome" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/22x22/Icon57.png</iconset>
   </property>
   <property name="text" >
    <string>Home</string>
   </property>
  </action>
  <action name="actionZoom" >
   <property name="icon" >
    <iconset resource="../resources.qrc" >:/icons/Toolbar/22x22/Icon58.png</iconset>
   </property>
   <property name="text" >
    <string>Zoom</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Shell</class>
   <extends>QsciScintilla</extends>
   <header>Shell.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc" />
 </resources>
 <connections/>
</ui>
